[2025-07-09T20:15:44.317+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T20:15:44.343+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_github_inference.run_model_inference manual__2025-07-09T20:15:25.333309+00:00 [queued]>
[2025-07-09T20:15:44.351+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_github_inference.run_model_inference manual__2025-07-09T20:15:25.333309+00:00 [queued]>
[2025-07-09T20:15:44.351+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-09T20:15:44.366+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_model_inference> on 2025-07-09 20:15:25.333309+00:00
[2025-07-09T20:15:44.380+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=287) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-09T20:15:44.381+0000] {standard_task_runner.py:63} INFO - Started process 290 to run task
[2025-07-09T20:15:44.380+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'daily_github_inference', 'run_model_inference', 'manual__2025-07-09T20:15:25.333309+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/daily_github_inference.py', '--cfg-path', '/tmp/tmpan_h1ckx']
[2025-07-09T20:15:44.382+0000] {standard_task_runner.py:91} INFO - Job 56: Subtask run_model_inference
[2025-07-09T20:15:44.394+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-09T20:15:44.417+0000] {task_command.py:426} INFO - Running <TaskInstance: daily_github_inference.run_model_inference manual__2025-07-09T20:15:25.333309+00:00 [running]> on host 49b9a4a3b242
[2025-07-09T20:15:44.501+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rajat' AIRFLOW_CTX_DAG_ID='daily_github_inference' AIRFLOW_CTX_TASK_ID='run_model_inference' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T20:15:25.333309+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T20:15:25.333309+00:00'
[2025-07-09T20:15:44.502+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T20:15:44.519+0000] {logging_mixin.py:188} INFO - [INFO] Running inference for timestamp: 2025-07-08-15
[2025-07-09T20:15:44.558+0000] {credentials.py:567} INFO - Found credentials in environment variables.
[2025-07-09T20:15:45.793+0000] {credentials.py:567} INFO - Found credentials in environment variables.
[2025-07-09T20:15:47.678+0000] {credentials.py:567} INFO - Found credentials in environment variables.
[2025-07-09T20:15:48.655+0000] {logging_mixin.py:188} INFO - [INFO] Inference complete. Output saved to s3: features/actor_predictions_2025-07-08-15.parquet
[2025-07-09T20:15:48.657+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-09T20:15:48.658+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T20:15:48.667+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=daily_github_inference, task_id=run_model_inference, run_id=manual__2025-07-09T20:15:25.333309+00:00, execution_date=20250709T201525, start_date=20250709T201544, end_date=20250709T201548
[2025-07-09T20:15:48.734+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T20:15:48.791+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T20:15:48.796+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
